web-client/src/store/index.ts

```diff
@@ -0,0 +1,18 @@
+import { combineReducers, createStore } from "redux";
+import { composeWithDevTools } from "redux-devtools-extension";
+import { exampleReducer } from './example/reducers'
+
+const rootReducer = combineReducers({ exampleReducer })
+
+export type AppState = ReturnType<typeof rootReducer>;
+
+export default function configureStore() {
+    // const middlewares = [thunkMiddleware];
+
+    const store = createStore(
+        rootReducer,
+        composeWithDevTools()
```

Comment:
we can already leverage the DEV env flag in order to turn this on, not sure on the key, but you can investigate it on the react docs

Reply:
Yes, you're right, I'll add that! Thanks